egregoria/src/vehicles/systems.rs

```diff
@@ -331,6 +361,11 @@ pub fn calc_decision<'a>(
         return default_return
     );
 
+    // If unparking, just set desired speed to 1/5 of usual
+    if let VehicleState::ParkedToRoad(_) = vehicle.state {
```

Comment:
you can use the matches! macro instead of deconstructing which copies the state (in theory, not sure since LLVM has good optimizations).

Reply:
and I know it probably doesn't matter a lot but this is a super hot loop so every bit can make a difference

Reply:
Doesn't matter anymore, since ParkedToRoad has no state now